packages/perseus/src/components/tex-buttons.tsx

```diff
@@ -1,3 +1,4 @@
+import {Keys as Key} from "@khanacademy/math-input";
```

Comment:
**suggestion**: This is a bit of a red flag. Aliasing imports often trips future folks up because when they encounter this symbol in this file, it's not the same as what's exported from `math-input`.  It's super minor, but these types of paper cuts add up in our codebase.

Reply:
In another PR, Ned said he preferred "Key" over "Keys" for types. I might have misunderstood what he was asking though. I'll change to "Keys." 🤷 I don't have a strong preference, but agree aliasing should be avoided when possible.